### DISCUSSION TITLE
toggle osd-bar on/off

### DISCUSSION THREAD

**Discussion by pjhfggij** [mpv-player/mpv#15254] [Q&A] [answered]
o and P seem to be by default bound to SHOW osd-bar (bar showing playback time), but it always disappears after osd-duration, however it would be helpful if osd-bar could stay up UNTIL user switches it off. Is that possible?

**Comment by guidocella** [comment, marked as answer]
That's not properly supported, but you can bind `cycle-values osd-duration 1000 3600000; show-progress`.

**Reply by pjhfggij** [reply]
that's brilliant for me, thank you. I was able to add it to `o` key, which was already bound to showing this bar. I wasn't sure if this will overwrite previous binding, but fortunately it seems like it only added new function. I've swapped values from your example so that the first time you press it, it uses the longer values and so seems to be "on" and next time "off" (100ms)
```
o cycle-values osd-duration 3600000 100; show-progress
```
